packages/vite/src/node/optimizer/index.ts

```diff
@@ -1,4 +1,5 @@
 import fs from 'node:fs'
+import { stat as fsStat, readdir } from 'node:fs/promises'
```

Comment:
Maybe we should use `import fsp from` instead to be consistent with the above fs import

Reply:
done both 👍🏼